Framework/Beamline/test/ComponentInfoTest.h

```diff
@@ -259,8 +265,8 @@ class ComponentInfoTest : public CxxTest::TestSuite {
 
     TS_ASSERT_THROWS(ComponentInfo(detectorsInSubtree, detectorRanges,
                                    componentsInSubtree, componentRanges,
-                                   parentIndices, positions, rotations,
-                                   scaleFactors, -1, -1),
+                                   parentIndices, isVisible, positions,
+                                   rotations, scaleFactors, -1, -1),
                      std::invalid_argument &);
   }
 
```

Comment:
You've fixed the builds to make them pass, but haven't added any tests. Would expect to see test methods for `ComponentInfo.isVisible()` ensuring that everything is wired up correctly.